cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -24,21 +27,61 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
                 component: 'component'
             };
 
+            xblockAncestorInfo = {
+                ancestors: [
+                    {
+                        category: 'vertical',
+                        display_name: 'unit_display_name_0',
+                        id: 'unit_ID_0'
+                    },
+                    {
+                        category: 'sequential',
+                        display_name: 'subsection_display_name_0',
+                        id: 'subsection_ID_0'
+                    },
+                    {
+                        category: 'chapter',
+                        display_name: 'section_display_name_0',
+                        id: 'section_ID_0'
+                    },
+                    {
+                        category: 'course',
+                        display_name: 'Demo Course',
+                        id: 'COURSE_ID_101'
+                    }
+                ]
+            };
+
             beforeEach(function() {
-                setFixtures(
-                    "<div class='breadcrumb-container'></div><div class='xblock-list-container'></div>"
-                );
                 TemplateHelpers.installTemplates([
-                    'move-xblock-list',
-                    'move-xblock-breadcrumb'
+                    'basic-modal',
```

Comment:
we needed this because not all things were loaded correctly, also this was used previously in `move_xblock_modal_spec.js`